### ISSUE TITLE
Connection refused | can't find any relevant logs

### ISSUE THREAD

**Issue by zachdaniel** [liveviewnative/liveview-client-swiftui#40]
I've followed the tutorial (for the most part, only the name of the app changed) and everything *seems* like it should be okay, but I'm seeing this in the output panel in XCode

```
2022-09-08 16:30:48.822624-0400 AshHq[44093:19117824] [LiveViewCoordinator] Connecting to http://localhost:4000/cats
2022-09-08 16:30:48.889501-0400 AshHq[44093:19118202] [connection] nw_socket_handle_socket_event [C1.1:2] Socket SO_ERROR [61: Connection refused]
```

Connecting to the page in the browser works fine (I'm on the `<text> Hello world </text>` step, very early on).

Simulating: iPhone 13 Pro
OS: Mac OS Monterey, apple silicon
Phoenix: 1.6.12
LiveView: 0.17.11

Happy to provide any other details! Super excited about this project!

**Comment by zachdaniel** [comment]
Oh, and in the liveview itself: 
```elixir
[info] GET /cats
[debug] Processing with LvnTutorialWeb.CatsListLive.Elixir.LvnTutorialWeb.CatsListLive/2
  Parameters: %{}
  Pipelines: []
[info] Sent 200 in 761µs
```

**Comment by zachdaniel** [comment]
The fact that it all works fine when rendering in the browser tells me that its probably something on the Swift side?

**Comment by zachdaniel** [comment]
![CleanShot 2022-09-08 at 16 43 12](https://user-images.githubusercontent.com/5722339/189222108-4d049695-3471-456d-b37e-c9b8b5ddb0ad.png)
The simulator shows "Initial Parse Error"

**Comment by zachdaniel** [comment]
Okay, nevermind, this is on me :) (as expected). I just peeked at the html that is rendering and its getting wrapped in `<main class="container">` because instead of following the tutorial I decided to try to tack it onto an existing app 😆

**Comment by zachdaniel** [comment]
Moral of the story here, make sure things are using the proper layouts.

**Comment by zachdaniel** [comment]
🤔 maybe I'm wrong. I've fixed that issue and now I only see the `<text>....` when viewing in the browser, and have the same error as listed above.

**Comment by zachdaniel** [comment]
Alright, confirmed that following the tutorial fresh works just fine, so this is clearly something on my end. Let this be a lesson to people who try to do what I did without trying the tutorial as intended first 😆

**Comment by zachdaniel** [comment]
Potential action item: on the initialParseError, display or log what failed to parse

**Comment by bcardarella** [comment]
if the element you are using in the template isn't a support SwiftUI component it will just not render anything

**Comment by zachdaniel** [comment]
Alright, I nailed it down. I was using some assigns in my `root` layout that caused an error because I was pulling them from browser session, a plug I hand't included in the pipeline for mobile. So I just made the root layout render `<%= @inner_content %>` like the other one (a misunderstanding of how all that was supposed to work). The step it was actually failing on was finding the csrf token on a meta tag 😆 . Either way, I'm up and running now 🔥 .
